server/sonar-server/src/main/java/org/sonar/server/db/migrations/MigrationStepModule.java

```diff
@@ -73,78 +73,77 @@
 import org.sonar.server.db.migrations.v52.FeedProjectLinksComponentUuid;
 import org.sonar.server.db.migrations.v52.MoveProjectProfileAssociation;
 
-import java.util.List;
+public class MigrationStepModule extends Module {
+  @Override
+  protected void configureModule() {
```

Comment:
:new: ![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)Protected method 'configureModule(...)' is never used.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUnusedProtectedMethod)